MidCamp/midcamp
# Lando

# Description

To reduce barriers to contribution, introducing support for [Lando](https://lando.dev/) local development tooling.  Work includes:

- Lando config
- Drush 10
- Swtich from `drupal-scaffold` to `core-composer-scaffold`
- Include [`amazeeio/drupal_integrations`](https://github.com/amazeeio/drupal-integrations)
- Converts existing Drush aliases for compatibility with Drush 10 and `amazeeio/drupal_integrations`
- Updates some Contrib modules for Drush 10 support
- Corrects [Drush site wide command paths](https://docs.drush.org/en/9.x/commands/#site-wide-drush-commands) in `composer.json`
- Updates CircleCI to use a more recent image (with newer PHP versions)

Existing Pygmy workflow is not altered, so users may opt to continue working that way if they desire.

# Test instructions

- Install Lando (version `v3.0.23` tested and confirmed working, YMMV with anything earlier)
- Run `lando start`
- Fetch a local database with `lando get-db`
- Fetch files with `lando get-files`
- Navigate to http://midcamp-org.lndo.site/ and observe you have a functional local environment.
